docs/en/observability/synthetics-reference/lightweight-config/common.asciidoc

```diff
@@ -234,6 +234,78 @@ By default, monitors are automatically retested if the monitor goes from "up" to
 retest_on_failure: false
 ----
 
+////////////////////////
+locations
+////////////////////////
+| [[monitor-locations]] *`locations`*
+(list of https://github.com/elastic/synthetics/blob/{synthetics_version}/src/locations/public-locations.ts#L28-L37[`SyntheticsLocationsType`])
+a| Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
```

Comment:
I'm not hyper aggressive about passive voice, but in this case maybe removing it makes the statement more concise?
```suggestion
a| Where to deploy the monitor. You can deploy monitors in multiple locations to detect differences in availability and response times across those locations.
```